ci/publish-artifactory.sh

```diff
@@ -37,6 +37,17 @@ push daml-script-runner $SCRIPT_RUNNER.asc
 push non-repudiation $NON_REPUDIATION
 push non-repudiation $NON_REPUDIATION.asc
 
+NON_REPUDIATION_CLIENT_JAR=non-repudiation-client-$RELEASE_TAG.jar
+NON_REPUDIATION_CLIENT_POM=non-repudiation-client-$RELEASE_TAG.pom
+NON_REPUDIATION_CLIENT_SRC=non-repudiation-client-$RELEASE_TAG-sources.jar
+
+push connect-ee-mvn/com/daml $NON_REPUDIATION_CLIENT_JAR
+push connect-ee-mvn/com/daml $NON_REPUDIATION_CLIENT_JAR.asc
```

Comment:
I copied this from above, but I can't see where these files are created and I don't see them locally when I `bazel build` the relative target. Should these stay here?

Reply:
They are created here https://github.com/digital-asset/daml/blob/45c4ba2230854e048027c999d476b3b85f5393d1/azure-pipelines.yml#L92

Reply:
As per @cocreature's [approval](https://github.com/digital-asset/daml/pull/9282#pullrequestreview-623962794), it appears that these are safe to stay.